eng/common/scripts/check-spelling-in-changed-files.ps1

```diff
@@ -286,13 +286,8 @@ if (!(Test-Path $CspellConfigPath)) {
 # Lists names of files that were in some way changed between the
 # current $SourceBranch and $TargetBranch. Excludes files that were deleted to
 # prevent errors in Resolve-Path
-Write-Host "git diff --diff-filter=d --name-only --relative $TargetBranch $SourceBranch"
-$changedFilesList = git diff `
-  --diff-filter=d `
-  --name-only `
-  --relative `
-  $TargetBranch `
-  $SourceBranch
+$gitDiffChanges = Join-Path $PSScriptRoot "git-diff-changes.ps1"
```

Comment:
We're already importing `common.ps1` in [here](https://github.com/Azure/azure-sdk-for-java/blob/main/eng/common/scripts/check-spelling-in-changed-files.ps1#L274). The cmdlet should just be available in `common.ps1` so we don't have to import it everywhere it's used.